CurrencyAPI/CurrencyAPI/Program.cs

```diff
@@ -1,17 +1,82 @@
+using System.Text.Json;
+
 namespace CurrencyAPI
 {
+    public class Bpi
```

Comment:
Ideally, those classed should be located in the separate file, I suggest: BitcoinAPISchema.cs